src/pocketmine/plugin/PluginDescription.php

```diff
@@ -97,6 +100,7 @@ private function loadMap(array $plugin){
 
 		$this->api = array_map("\strval", (array) ($plugin["api"] ?? []));
 		$this->compatibleMcpeProtocols = array_map("\intval", (array) ($plugin["mcpe-protocol"] ?? []));
+		$this->compatibleOperatingSystems = array_map("\strval", (array) ($plugin["operating-systems"] ?? []));
```

Comment:
I was under the impression we were using `os` for this key. Maybe you lost some changes when you force-pushed.